### ISSUE TITLE
ValueError: zero-size array to reduction operation fmin which has no identity

### ISSUE THREAD

**Issue by lauri-elevant** [konstantint/PassportEye#34]
In this commit:
https://github.com/konstantint/PassportEye/commit/e388a973d3b3fc289c4939a43641f712f18f4100

the try/except block was removed from `mrz/scripts.py`. With 1.4.0, we get occasional errors that were probably ignored before, just returning "no match" silently. I am not sure if it indicates a deeper underlying issue that could be fixed and the accuracy thereby raised, or it is something that really ought to be silently ignored.

Traceback is:
```
filename, mrz_, walltime = process_file((args.filename, args.save_roi is not None, extra_params))
File "/usr/local/lib/python3.6/site-packages/passporteye/mrz/image.py", line 227, in __call__
File "/usr/local/lib/python3.6/site-packages/passporteye/mrz/scripts.py", line 34, in process_file
File "/usr/local/lib/python3.6/site-packages/passporteye/util/pipeline.py", line 102, in __getitem__
new_mrz = __pipeline__['mrz']
text = ocr(roi, extra_cmdline_params=self.extra_cmdline_params)
File "/usr/local/lib/python3.6/site-packages/numpy/lib/nanfunctions.py", line 292, in nanmin
ValueError: zero-size array to reduction operation fmin which has no identity
```

Apologies for the traceback being garbled, our logging solution seems to have logged individual lines of output in undefined order.

**Comment by konstantint** [comment]
I could not reproduce it, but I suspect the error came from a spot in the code which assumed that a hypothetical "box" located by one of the pipeline steps necessarily has a non-zero shape. I added an explicit check, so that hopefully you won't get that error anymore. Let me know if it still appears. The fix is uploaded to PyPI as V1.4.1.

**Comment by lauri-elevant** [comment]
It only reproduces on some really odd documents, which might even not be passports - in DreamApply use the library not only to extract data from passports, but also put all uploads through it to catch any mis-tagged uploads. We are essentially fuzzing the library.

Good to know it is fixed. Thank you!
